dali/operators/decoder/audio/audio_decoder_op.cc

```diff
@@ -30,6 +30,9 @@ This operator produces two outputs:
 )code")
   .NumInput(1)
   .NumOutput(2)
+  .OutputDoc(0, "decoded", "TensorList of int16, int32 or float", "The decoded audio recordings.")
```

Comment:
I thought the conclusion was to start using `TensorList` for describing outputs. Wasn't it?